hack/violation_exception_v2alpha1.list

```diff
@@ -0,0 +1,21 @@
+API rule violation: list_type_missing,github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v2alpha1,ContainerOverride,Args
+API rule violation: list_type_missing,github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v2alpha1,ContainerOverride,Command
+API rule violation: list_type_missing,github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v2alpha1,ContainerOverride,Env
+API rule violation: list_type_missing,github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v2alpha1,ContainerOverride,EnvFrom
+API rule violation: list_type_missing,github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v2alpha1,ContainerOverride,VolumeMounts
+API rule violation: list_type_missing,github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v2alpha1,DatasetConfig,Env
```

Comment:
Why do we have those violations ?

Reply:
This is automatically generated. So, I guess @varshaprasad96 does not know the reason.

Reply:
Yeah, these are auto-generated. Though, some of this is because of the OpenAPI Schema validation and _could possibly_ be fixed by adding appropriate CRD markers from KB. I'll dig deeper into this next week, and create a PR to fix them if possible.

Reply:
Sure, thank you!

Reply:
Fixed in https://github.com/kubeflow/training-operator/pull/2317